.ci/install_cri_containerd.sh

```diff
@@ -25,10 +27,16 @@ source "${script_dir}/lib.sh"
 CONTAINERD_OS=$(go env GOOS)
 CONTAIENRD_ARCH=$(go env GOARCH)
 
-cri_containerd_version=$(get_version "externals.cri-containerd.version")
+cri_containerd_tarball_version=$(get_version "externals.cri-containerd.version")
 cri_containerd_repo=$(get_version "externals.cri-containerd.url")
 
-source /etc/os-release || source /usr/lib/os-release
+echo "Get cri_containerd version"
+go get "${containerd_repository}"
+cri_containerd_version=$(
+	cd "${GOPATH}/src/${containerd_repository}";
+	git checkout "v${cri_containerd_tarball_version}" >&2;
+	cat vendor.conf | grep "github.com/containerd/cri" | awk '{print $2}';
```

Comment:
You could remove the `cat` using:

```sh
grep "github.com/containerd/cri" vendor.conf | awk '{print $2}';
```

Also, it would be safer to anchor that string to ensure you only get one match:

```sh
grep -E "^github.com/containerd/cri\>" vendor.conf | awk '{print $2}';
```

But also you can use the variable making the potential final version:

```sh
grep -E "^${cri_repository}\>" vendor.conf | awk '{print $2}';
```

Reply:
thanks!!! a lot to learn~~~😘